var/spack/repos/builtin/packages/openfoam-com/package.py

```diff
@@ -659,6 +663,10 @@ def __init__(self, spec, **kwargs):
         # spec.architecture.target is like `uname -m`
         target   = spec.architecture.target
 
+        # Platform detects as 'cray' - treat like 'linux'
+        if platform == 'cray':
+            platform = 'linux'
```

Comment:
If our initial mileage is ok with this the simple approach, I would go for that. It has a much lower maintenance overhead and I want to see what happens when we start dealing with additional processor types (eg, ARM) before rejigging this again. I suspect that I might need even more work there soon enough.